packages/cashier/src/stores/account-transfer-store.ts

```diff
@@ -85,17 +67,18 @@ export default class AccountTransferStore {
     }
 
     @action.bound
-    setBalanceByLoginId(loginid, balance) {
-        this.accounts_list.find(acc => loginid === acc.value).balance = balance;
+    setBalanceByLoginId(loginid: string | undefined, balance: string | undefined) {
+        const account = this.accounts_list.find(acc => loginid === acc.value);
+        if (account) account.balance = balance;
     }
 
     @action.bound
-    setBalanceSelectedFrom(balance) {
+    setBalanceSelectedFrom(balance: string | undefined): void {
         this.selected_from.balance = balance;
```

Comment:
I'd like to suggest delete 'undefined' here and below in the code.
I believe we shouldn’t call this function if the value is 'undefined'. So it’s better to add check before calling this function.
Also please check the type of balance. As I see 'selected_from' has type TAccount, where balance can be 'string' or 'number'. 
so please check what is the correct type for balance

Reply:
@yauheni-kryzhyk-deriv Nice tip. Changed.